File: app/src/main/java/ai/elimu/content_provider/room/dao/LetterSound_LetterDao.java
package ai.elimu.content_provider.room.dao;

import androidx.room.Dao;
import androidx.room.Insert;
import androidx.room.Query;

import ai.elimu.content_provider.room.entity.LetterSound_Letter;

@Dao
public interface LetterSound_LetterDao {

    @Insert
    void insert(LetterSound_Letter letterSound_Letter);

    @Query("DELETE FROM LetterSound_Letter")
    void deleteAll();
}

File: app/src/main/java/ai/elimu/content_provider/room/dao/LetterSound_LetterDao.kt
package ai.elimu.content_provider.room.dao

import ai.elimu.content_provider.room.entity.LetterSound_Letter
import androidx.room.Dao
import androidx.room.Insert
import androidx.room.Query

@Dao
interface LetterSound_LetterDao {
    @Insert
    fun insert(letterSound_Letter: LetterSound_Letter)

    @Query("DELETE FROM LetterSound_Letter")
    fun deleteAll()
}
